CMakeLists.txt

```diff
@@ -287,15 +349,13 @@ if(ACTS_BUILD_PLUGIN_DD4HEP)
 endif()
 if(ACTS_BUILD_PLUGIN_JSON)
   if(ACTS_USE_SYSTEM_NLOHMANN_JSON)
-    message(STATUS "Using system installation of nlohmann::json")
     find_package(nlohmann_json ${_acts_nlohmanjson_version} REQUIRED CONFIG)
```

Comment:
referring to my comment here https://github.com/acts-project/acts/pull/3099#discussion_r1601010108